fastlane/spec/actions_specs/set_build_number_repository_spec.rb

```diff
@@ -22,6 +22,16 @@
 
         expect(Fastlane::Actions.lane_context[Fastlane::Actions::SharedValues::BUILD_NUMBER]).to match(/cd .* && agvtool new-version -all asd123/)
       end
+      
+      it "set build number with explicit xcodeproj" do 
+        result = Fastlane::FastFile.new.parse("lane :test do
+            set_build_number_repository(
+              xcodeproj: asd123/project.xcodeproj
```

Comment:
@mfurtak I'm not sure what the path of the xcodeproj actually is, but other than that I think this test is good